### ISSUE TITLE
Invalid hexadecimal escape sequence

### ISSUE THREAD

**Issue by divinebovine** [sequelize/sequelize-auto#476]
We are using Postgresql 11 and have a bytea column. When we generate our ORM it creates the definition for our column:

```js
some_column: {
      type: "BYTEA",
      allowNull: false,
      defaultValue: "\x"
    }
```
When we try to use this code, we run into the following error:

```sh
/opt/app/node_modules/@simple/some-orm/models/someTable.js:45
      defaultValue: "\x"
                     ^^^

SyntaxError: Invalid hexadecimal escape sequence
    at wrapSafe (internal/modules/cjs/loader.js:1053:16)
    at Module._compile (internal/modules/cjs/loader.js:1101:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/opt/app/node_modules/@simple/some-orm/models/init-models.js:26:17)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
```

Should the default value actually be "\\x"? I'm aware postgresql uses "\x" to escape a hex string, but maybe its falling afoul of javascript string escapes.

**Comment by steveschmitt** [comment]
That's bad!  Thanks for reporting this.

What does the column definition look like in the Postgres database?  What is the default value?

**Comment by divinebovine** [comment]
here's an edited output from psql:
```sh
                                           Table "public.some_table"
        Column        |            Type             | Collation | Nullable |             Default
----------------------+-----------------------------+-----------+----------+----------------------------------
 id                   | integer                     |           | not null | generated by default as identity
 some_column          | bytea                       |           | not null | '\x'::bytea
Indexes:
    "pk_user_sync" PRIMARY KEY, btree (id)
```

**Comment by divinebovine** [comment]
I'm wondering if the default value should actually be escaped like "\\x" in sequelize auto.

**Comment by divinebovine** [comment]
I have tested locally by replacing the `\x` with `\\x` and it seems to work fine. It will just be a matter of escaping what the database reports.

**Comment by steveschmitt** [comment]
Fixed in 0.7.7
